packages/@adobe/spectrum-css-temp/components/table/index.css

```diff
@@ -129,9 +135,32 @@ governing permissions and limitations under the License.
   font-weight: var(--spectrum-table-cell-text-font-weight);
   /* need to subtract 1px here because 14px * 1.5 + 14px + 14px = 49px, and we want 48px */
   line-height: calc(calc(var(--spectrum-table-cell-text-size) * var(--spectrum-table-cell-text-line-height)) - 1px);
-  padding: var(--spectrum-table-cell-padding-y) var(--spectrum-table-cell-padding-x);
-  min-height: var(--spectrum-table-cell-min-height);
+  padding: var(--spectrum-table-cell-regular-padding-y) var(--spectrum-table-cell-padding-x);
+}
+
+.spectrum-Table--compact .spectrum-Table-cell {
+  padding-top: var(--spectrum-table-cell-compact-padding-y);
+  padding-bottom: var(--spectrum-table-cell-compact-padding-y);
+}
+
+.spectrum-Table--spacious .spectrum-Table-cell {
+  padding-top: var(--spectrum-table-cell-spacious-padding-y);
+  padding-bottom: var(--spectrum-table-cell-spacious-padding-y);
+}
+
+.spectrum-Table-cellContents {
+  /* truncate text with ellipsis */
   overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+
+  /* allow focus ring of child to extend outside the bounds */
```

Comment:
cringe....
there has to be a better way to make sure we apply this to everything that is a container :(
I'm not sure how to document this too so that when people want to make their own overflow hidden containers, they don't miss this...

I know we're not going to solve it in this PR